code/__HELPERS/logging.dm

```diff
@@ -82,6 +82,10 @@
 	if (config.log_pda)
 		diary << "\[[time_stamp()]]CHAT: [text]"
 
+/proc/log_comment(text)
+	if (config.log_pda) //reusing as not worth its own config option really

```

Comment:
- [ ] Keep as is?

- [ ] Update to correct standard?

- [ ] Update to correct standard across the board?